posthog/celery.py

```diff
@@ -260,6 +264,14 @@ def clickhouse_mark_all_materialized():
         mark_all_materialized()
 
 
+@app.task(ignore_result=True)
+def clickhouse_send_license_usage():
+    if is_clickhouse_enabled() and settings.EE_AVAILABLE:
```

Comment:
```suggestion
    if is_clickhouse_enabled():
```

`is_clickhouse_enabled` already checks `settings.EE_AVAILABLE`